### PR TITLE
create index for ckb transaction to optimize performance

### PR DESCRIPTION
**Pull request by ShiningRay**
Some addresses contains too many transactions ( > 100000), cause pagination very slow.
I create a new union index to cover filtering and ordering to optimize that performance.

### CODE REVIEW

File: db/schema.rb [hunk 1]
```diff
@@ -418,6 +420,46 @@
     t.index ["table_name", "count"], name: "index_table_record_counts_on_table_name_and_count"
   end
 
+  create_table "token_collections", force: :cascade do |t|
```
Comment (Keith-CY) [inline db/schema.rb:423]: These codes are not related to the issue

**Review by zmcNotafraid — APPROVED** [review]
